src/tod_manager.cpp

```diff
@@ -49,6 +52,11 @@ tod_manager::tod_manager(const config& scenario_cfg):
 		random_tod_ = false;
 
 	time_of_day::parse_times(scenario_cfg,times_);
+	liminal_bonus_ = calculate_best_liminal_bonus(times_);
```

Comment:
I think this line would be better placed as an `else` clause in the following `if` statement, since if the `if` is entered the calculation isn't even used.

Reply:
if the if is entered BUT the attribute contains something that is not a number, the calculation is used.

Reply:
Ah, so what you'd need to move this to an else is a way to test the type of the attribute. Not sure if that exists.